src/rpc/types.ts

```diff
@@ -12,11 +12,12 @@ import type {
 } from './callers';
 import type { NodeId } from '../nodes/types';
 import type { Certificate } from '../keys/types';
+import type { POJO } from '../types';
 
 /**
  * This is the JSON RPC request object. this is the generic message type used for the RPC.
  */
-type JsonRpcRequestMessage<T extends JSONValue = JSONValue> = {
+type JSONRPCRequestMessage<T extends JSONValue = JSONValue> = {
   // A String specifying the version of the JSON-RPC protocol. MUST be exactly "2.0"
```

Comment:
I think if you want these comments to show up in the typedoc generated comments, all of these must be docblocks, not `//` comments.

We only use line comments for inside functions/procedures.

Reply:
They're mostly for reference and explaining what each field is for. It doesn't need to show up in the IDE docs.

Reply:
Not the IDE docs, the typedocs. If it's switched to docblocks, you should be able to see it too.

Reply:
I've made them doc comments and some to the message types.